packages/react-ai/src/components/AIConversation/AIConversation.tsx

```diff
@@ -19,7 +19,7 @@ interface _AIConversationProps
   extends AIConversationProps,
     AIConversationInput {}
 
-export const AIConversation = ({
+const AIConversationBase = React.forwardRef(function AIConversation({
```

Comment:
Don't see a forwardRef used?

Reply:
Good call, remove, i just copied the format for storagemanager